docs/creating-manifest/basic-configs.md

```diff
@@ -385,6 +395,159 @@ The service starts if:
+    * The *mongodb* cluster is created as **[replica-set](https://jelastic.com/blog/mongodb-replica-set-master-slave-failover/)** with topology that comrises tree nodes one *Primary* and two *Secondary* nodes.      
+   
+For example:  
+
+@@@
+```yaml
+type: install
+name: Auto-Cluster
+
+nodes:
+  - nodeType: payara
+    cluster: true
+```
+```json
+{
+  "type": "install",
+  "name": "Auto-Cluster",
+  "nodes": [
+    {
+      "nodeType": "payara",
+      "cluster": true
+    }
+  ]
+}
+```
+@@!  
+  
+  
+![autoclustering-switch](/img/autoclustering-mysql.png)  
+ 
+  * *object* - this is applicable for *mysql*/*mariadb* `nodeType` only. Object contains multiple options can be passed as configuration parameters:   
+    * `scheme` *[optional]* - configures database [replication scheme](https://jelastic.com/blog/mysql-mariadb-database-auto-clustering-cloud-hosting/) for:  
+      * `mysql` - **slave** (Master-Slave), **master** (Master-Master), **single** (Single Primary Group Replication), **multi** (Multi Primary Group Replication)  
+      * `mariadb` - **slave** (Master-Slave), **master** (Master-Master), **galera** (Galera Cluster)  
+    * `is_proxysql` *[optional][boolean]* - *'true'* adds a **proxysql** load balancer layer to the topology and configures it as an entry point to the database cluster  
+    * `db_user` *[optional]* - sets up a database username. If not defined the system will generate one by default  
+    * `db_pass` *[optional]* - sets up a password for `db_user`. If not defined the system will generate one by default  
+  
+A [cluster configuration object](basic-configs/autoclustering) should be passed to the `cluster` field to enable custom auto-clustering
+
+*Master-Master* replication topology with ProxySQL node as the entry point:  
+  
+@@@
+```yaml
+type: install
+name: Auto-Cluster
+
+nodes:
+  - nodeType: mysql
+    cluster:
+      scheme: master
+      is_proxysql: true
+      db_user: "mydbuser"
+      db_pass: "mydbpasswd"
+```
+```json
+{
+  "type": "install",
+  "name": "Auto-Cluster",
+  "nodes": [
+    {
+      "nodeType": "mysql",
+      "cluster": {
+        "scheme": "master",
+        "is_proxysql": true,
+        "db_user": "mydbuser",
+        "db_pass": "mydbpasswd"
+      }
+    }
+  ]
+}
+```
+@@!  
+
+Once the cluster parameters were applied to respective layer, you won't be able to change them.
+
+### isDeploySupport
+
+You can invoke it through the API method environment.control.ApplyNodeGroupData.
+
+For example:
+
+```
+https://app.jelastic.com/1.0/environment/control/rest/applynodegroupdata?data={"isDeploySupport":false}&envName=env-1&session=1dx1d2eb8c6b29cddfb28cfd4e93f80c15c&nodeGroup=cp  
+```
+In case deploy is not supported an error code is logged: *DEPLOY_IS_NOT_SUPPORTED = 2550  *
```

Comment:
Необходимо перенести этот блок после https://docs.cloudscripting.com/creating-manifest/basic-configs/#skip-node-emails, также:
* переименовать в Update Node Group settings
* поправить текст 
You can update node group settings through the API method [environment.control.ApplyNodeGroupData](https://docs.jelastic.com/api/#!/api/environment.Control-method-ApplyNodeGroupData).
* удалить In case deploy is not supported an error code is logged: *DEPLOY_IS_NOT_SUPPORTED = 2550  
* добавить якорь для перехода с блоков 'The setting is stored in *nodeGroup* settings and can be overridden via API'

Reply:
исправил https://github.com/vlobzakov/docs/commit/e334ba903ed72e3d2d6a04e32a2edbceb017381d#diff-8579cb093488ca3b1e53aa719d602435